test/unit/toUint8Array.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const assert = require('assert');
+
+const sharp = require('../../');
+const fixtures = require('../fixtures');
+
+describe('toUint8Array', () => {
+  beforeEach(function () {
+    sharp.cache(false);
+  });
+  afterEach(function () {
+    sharp.cache(true);
+  });
+
+  it('should output data as Uint8Array', (done) => {
+    const image = sharp(fixtures.inputJpg320x240);
+    image.toUint8Array().then((data) => {
+      assert.strictEqual(data instanceof Uint8Array, true);
```

Comment:
Node.js `Buffer` objects will always satisfy the `instanceof Uint8Array` check. Perhaps verify `data.constructor.name` instead?

Reply:
Used your proposed way, worked like a charm